ADOPTERS.md

```
@@ -0,0 +1,19 @@
+> Do you want to add yourself to this list? Simply fork the repository and open a PR with the required change.
+> We have a short description of the adopter types at the bottom of this page. Each adopter type is in alphabetical order. 
+
+# NetObserv Adopters
+
+| Type | Name | Since | Website | Use-Case |
+|:-|:-|:-|:-|:-|
+| Vendor | Red Hat, Inc. | 2023 | [link](https://www.redhat.com) | Red Hat provides the Network Observability product for OpenShift, built from NetObserv, and uses it internally. |
+| Integration | ovn-kubernetes | 2023 | [link](https://ovn-kubernetes.io/) | OVN-Kubernetes is a CNCF project having several points of integration with NetObserv. |
+| Integration | IBM SevOne | 2024 | [link](https://community.ibm.com/community/user/blogs/rupert-gregory/2024/06/19/kubernetes-observability-with-ibm-sevone) | SevOne uses NetObserv for Kubernetes network observability since 7.0. |
+
+### Adopter Types
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)